src/test/java/bio/terra/common/BaseTest.java

```diff
@@ -2,6 +2,8 @@
 
 import static org.junit.Assert.assertEquals;
 
+import bio.terra.cloudevents.GCSEvent;
+import bio.terra.cloudfunctions.common.CloudStorageEventHarness;
 import bio.terra.cloudfunctions.common.MediaTypeWrapper;
 import bio.terra.cloudfunctions.common.StorageObjectEventHarness;
 import com.google.cloud.functions.Context;
```

Comment:
could the fixture data in this class - `MOCK_EVENT_GZIP`, `MOCK_EVENT_JSON`, `MOCK_STORAGE_OBJECT_DATASTREAM` - be defined as files under src/test/resources instead of hardcoded into this file?

Reply:
All hardcoded test fixture data are now replaced by test resource files.